index.html

```diff
@@ -60,9 +60,296 @@ <h2>I`m a sofware developer</h2>
+                     <li class="list-name">
+                       Ruby on rails
+                     </li>
+                     <li class="list-name">
+                       css 
+                     </li>
+                     <li class="list-name">
+                       javascript
+                     </li>
+                     <li class="list-name">
+                       html
+                     </li>
+                   </ul>
+                   <div class="bg" >
+                     <button class="goproject" type="button">See project</button>
+                   </div>
+                  </section>
+        
+                  <section class="card">
+                    <div class="img"></div>
+                    <h3 class="cards-title">Multi-Post Stories Gain+Glory</h3>
+                   <ul class="mini-card-list">
+                     <li class="list-name">
+                       Ruby on rails
+                     </li>
+                     <li class="list-name">
+                       css 
+                     </li>
+                     <li class="list-name">
+                       javascript
+                     </li>
+                     <li class="list-name">
+                       html
+                     </li>
+                   </ul>
+                   <div class="bg">
+                     <button class="goproject" type="button">See project</button>
+                   </div>
+                  </section>
+        
+               </section>
+
+
+            <section class="about-me">
+
+              <div class="backaboutone"><img src="/img/background-1-about-me.svg" alt=""></div>
+              <h2>About me</h2>
+              <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+              <div class="backabouttwo"><img src="/img/backgound-2-about-me.svg" alt=""></div>
+              
+              <div class="bg">
+                <button class="goproject" type="button">Get My Resume</button>
               </div>
+            
+            </section>
+
+            <ul>
+              <li class="tech-card">
+                <div class="img"></div>
+                <div class="about-me-icons">
+                  <div><img src="/Offline-Icons/about-me-icons.-1.svg" alt=""></div>
+                </div>
+      
+                <h3>Languages</h3>
+      
+                <ul class="mini-card-list-l">
+                  <li class="list-name-i">
+                    Javascript
+                  </li>
+                  <li class="list-name-i">
+                    Ruby
+      
+                  </li>
+                  <li class="list-name-i">
+                    Html
+                  </li>
+                  <li class="list-name-i">
+                    Css 
+                  </li>
+                </ul>
+      
+              </li>
```

Comment:
- [ **OPTIONAL** ]  I think it would be better :+1: :100:  to remove all unnecessary empty code lines to make your code more professional and smart. I suggest you apply for every unnecessary empty code line in your code